idaes/models/unit_models/packed_bed_reactor.py

```diff
@@ -0,0 +1,337 @@
+##############################################################################
```

Comment:
As a first observation, this might be better placed in `models_extra` for now. The requirements for getting code merged there are significantly lower than for the main `models` folder. This might be suitable for merging in `models`, but would documentation and thorough testing.